locust/main.py

```diff
@@ -350,10 +350,15 @@ def main():
         sys.exit(0)
 
     locustfile = find_locustfile(options.locustfile)
+
     if not locustfile:
         logger.error("Could not find any locustfile! Ensure file ends in '.py' and see --help for available options.")
         sys.exit(1)
 
+    if locustfile == "locust.py":
+        logger.error("The locustfile must not be named `locust.py`. Please rename the file and try again.")
```

Comment:
@justiniso thanks!
however, the file can be named something when you use the `-f` option.  do you think your message implies it must be named `locustfile.py`?